### PR TITLE
Fixed to use the emoticon for the up and down.

### PR DESCRIPTION
**Pull request by brandenr**
Changed to ignore retry comments so that it doesn't override whether or not it actually has been reviewed

### CODE REVIEW

**Comment by stephanheilner** [comment]
👍

**Comment by nickmshelley** [comment]
👍

File: index.html [hunk 1]
```diff
@@ -215,8 +215,8 @@ <h4 style="text-align: center;">OR</h4>
         
         <script>
             $(function() {
-                var approveSentinels = [ ':+1:', ':v:', ':metal:' ];
-                var rejectSentinels = [ ':-1:', ':poop:', ':facepunch:' ];
+                var approveSentinels = [ ':+1:', ':v:', ':metal:', '👍' ];
+                var ignoredSentinels = [ 'retry', 'Retry' ];
```
Comment (hiltonc) [inline index.html]: Might be better to make `hasSentinel` work case insensitively.

**Comment by nickmshelley** [comment]
Sorry @hiltonc. Didn't see your comment before the merge.

**Comment by nickmshelley** [comment]
Oh, you commented after the merge. I see.

File: index.html (hunk 1 above)
Comment (brandenr) [inline index.html]: I wasn't sure if you wanted that or not since PASS and FAIL seemed intentionally uppercase to avoid accidentally picking that up

**Comment by hiltonc** [comment]
You didn't see it before the merge? I'm disappointed.

File: index.html (hunk 1 above)
Comment (hiltonc) [inline index.html]: Oh. Good point.

**Pull request merged**